packages/node-resolve/src/index.js

```diff
@@ -30,6 +30,7 @@ const deepFreeze = object => {
   return object;
 };
 const defaults = {
+  exportConditions: ['module', 'default'],
```

Comment:
@lukastaegert if I understand correctly this is the approach webpack will take. this seems like a good default for rollup as well

Reply:
I would also add the `import` condition as a default because within Rollup, the assumption is that all modules are ESM. The only exception which is the commonjs plugin should receive its own logic IMO, or better, we should add an API to the node-resolve plugin that the commonjs plugin can access. But I would see this beyond this PR. Otherwise, either Rollup unnecessarily picks a default branch that is CommonJS or worse, it does not pick any branch at all when presented with the choice between `import` and `require`.

Another question: Why are you listing `default` as a condition? To my understanding it is a catch all that always applies.

Reply:
Good point about `import`, I'll add it in. 

In the way package entrypoints work, `default` is a condition like any other in a technical sense. We need to iterate the array in order and find the best match, ending up with default as the last. However, I understand the confusion since it's listed under the plugin options. We should just add it to the array in the code, a user should not be able to remove it.